static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -78,7 +80,7 @@ const FeedbackListItem = forwardRef<HTMLDivElement, Props>(
               <FeedbackItemUsername feedbackIssue={feedbackItem} detailDisplay={false} />
             </span>
           </TextOverflow>
-          <span style={{gridArea: 'time'}}>
+          <span style={{gridArea: 'time', textAlign: 'right'}}>
```

Comment:
Let's make a styled component for this and reserve inline styles for grid-area and dynamic styles